plugins/osdn/multitenant/multitenant.go

```diff
@@ -31,6 +31,9 @@ func Master(osClient *osclient.Client, kClient *kclient.Client, clusterNetwork s
 	if err != nil {
 		glog.Fatalf("SDN initialization failed: %v", err)
 	}
+	// TODO: Get admin namespaces from master node config
```

Comment:
@rajatchopra do we want to handle this now or postpone for later when there is a requirement?